hera_cal/lst_stack/averaging.py

```diff
@@ -657,16 +702,25 @@ def average_and_inpaint_simultaneously(
         stack, get_std=False, get_mad=False, inpainted_mode=False, mean_fill_value=0.0
     )
 
+    # Map polarizations to pol indices for correct noise variance computation
+    pol_to_idx = {}
```

Comment:
I'd clarify that this is mapping antenna polarizations (antpol / jpol) to visibility pol idx.

Reply:
Done